client/my-sites/plans/plan-overview/plan-features/feature.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import SectionHeader from 'components/section-header';
+
+const PlanFeature = React.createClass( {
```

Comment:
I'm not a big fan of mixing approaches likes this. It makes the code more difficult to read in my opinion.